command/agent/command.go

```diff
@@ -88,6 +100,25 @@ func (c *Command) readConfig() *Config {
 		return nil
 	}
 
+	// Split the servers.
+	if servers != "" {
+		cmdConfig.Client.Servers = strings.Split(servers, ",")
+	}
+
+	// Parse the meta flags.
+	if len(meta) != 0 {
+		cmdConfig.Client.Meta = make(map[string]string)
+		for _, kv := range meta {
+			parts := strings.Split(kv, "=")
```

Comment:
We can use `SplitN` here to allow `=` signs in the value. Might be useful for encoded values and such.